app/mailers/job_mailer.rb

```diff
@@ -6,9 +6,15 @@ class JobMailer < ApplicationMailer
   def completion_email
     @job_run = params[:job_run]
     @user = @job_run.batch_context.user
+
+    # This email is sent when either (1) a discovery report completes, or (2) a preassembly job ends with errors
+    # If the preassembly job completes with no errors (and items are accessioning), a separate email is instead sent
+    # when accessioning is complete.
+    @subject = "[#{@job_run.batch_context.project_name}] Your #{@job_run.job_type.humanize} job "
+    @subject += @job_run.job_type == 'preassembly' ? 'encountered an error' : 'completed'
     mail(
```

Comment:
We need some logic here because a discovery report still needs to say "completed" while this ticket changes the preassembly job to "encountered an error"